java/StreamingPipeInAndOut.java

```
@@ -0,0 +1,22 @@
+// Script reads text from a named pipe and writes it another named pipe, capitalized
+import java.io.*;
+
+public class StreamingPipeInAndOut {
+    @SuppressWarnings("resource")
```

Let's remove this `@SuppressWarnings("resource")` and instead close the output and input at the end of the function. I know we generally want to ignore exceptions, and it would be fine for these scripts not to close their input and output streams, but I think the having `input.close()` and `output.close()` at the end of the script is easier to understand than `@SuppressWarnings("resource")` for someone reading this code.